README.md

```diff
@@ -6,10 +6,10 @@ This is a mono repository, which provides all the JSON Schema documents for vali
 
 ## Overview
 
-* This repository contains [JSON Schema](https://json-schema.org) files for all the versions of AsyncAPI specification.
+* This repository contains [JSON Schema](https://json-schema.org) files for all the versions of AsyncAPI specification. There are two types of JSON Schema files, with and without **$id** feature 
```

Comment:
nitpick: I would suggest we add a clarification on why those new schemas (without $id` are meant to).